packages/transformers/js/src/visitors/dependencies.js

```diff
@@ -92,7 +94,34 @@ export default ({
           return;
         }
 
-        addDependency(asset, args[0], {isAsync: true});
+        let meta;
+        let importAttributesNode = args[1];
+        if (importAttributesNode != null) {
+          if (importAttributesNode.type !== 'ObjectExpression') {
+            throw new Error(
+              'Second argument to import() must be an object expression',
+            );
+          }
+          meta = {
+            importAttributes: objectExpressionNodeToJSONObject(
+              importAttributesNode,
+            ),
+          };
+
+          let newAttributesNode = withoutUnknownImportAttributes(
+            importAttributesNode,
+          );
+          if (newAttributesNode.properties.length > 0) {
+            // Replace the import attributes with only the allowed attribute(s)
+            args[1] = newAttributesNode;
+          } else {
+            // If no allowed attribute(s) are left, remove the attributes argument
+            // from the import() call entirely
+            args.splice(1, 1);
```

Comment:
Should we always do this? We replace import with require below, which doesn't accept a second argument.